sys/linux/dev_img_rogue.txt

```diff
@@ -0,0 +1,1925 @@
+	pui32SyncPMRFlags		ptr[in, array[int32]]
+	phSyncPMRs			ptr[in, array[IMG_HANDLE]]
+	ui32RenderTargetSize		int32
+	ui32NumberOfDrawCalls		int32
+	ui32NumberOfIndices		int32
+	ui32NumberOfMRTs		int32
+	ui64Deadline			int64
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXKICKTA3D2 {
+	hUpdateFence	PVRSRV_FENCE
+	hUpdateFence3D	PVRSRV_FENCE
+	eError		int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXPOPULATEZSBUFFER {
+	hsZSBufferKM	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXPOPULATEZSBUFFER {
+	hsPopulation	IMG_HANDLE
+	eError		int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXRENDERCONTEXTSTALLED {
+	hRenderContext	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXRENDERCONTEXTSTALLED {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXSETRENDERCONTEXTPRIORITY {
+	hRenderContext	IMG_HANDLE
+	ui32Priority	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXSETRENDERCONTEXTPRIORITY {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXSETRENDERCONTEXTPROPERTY {
+	hRenderContext	IMG_HANDLE
+	ui32Property	int32
+	ui64Input	int64
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXSETRENDERCONTEXTPROPERTY {
+	ui64Output	int64
+	eError		int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXUNPOPULATEZSBUFFER {
+	hsPopulation	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXUNPOPULATEZSBUFFER {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXCREATETRANSFERCONTEXT {
+	ui32Priority		int32
+	ui32FrameworkCmdize	len[psFrameworkCmd, int32]
+	psFrameworkCmd		ptr[in, array[int8]]
+	hPrivData		IMG_HANDLE
+	ui32PackedCCBSizeU8888	int32
+	ui32ContextFlags	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXCREATETRANSFERCONTEXT {
+	hTransferContext	IMG_HANDLE
+	hCLIPMRMem		IMG_HANDLE
+	hUSCPMRMem		IMG_HANDLE
+	eError			int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXDESTROYTRANSFERCONTEXT {
+	hTransferContext	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXDESTROYTRANSFERCONTEXT {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXSETTRANSFERCONTEXTPRIORITY {
+	hTransferContext	IMG_HANDLE
+	ui32Priority		int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXSETTRANSFERCONTEXTPRIORITY {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXSETTRANSFERCONTEXTPROPERTY {
+	hTransferContext	IMG_HANDLE
+	ui32Property		int32
+	ui64Input		int64
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXSETTRANSFERCONTEXTPROPERTY {
+	ui64Output	int64
+	eError		int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXSUBMITTRANSFER2 {
+	hTransferContext		IMG_HANDLE
+	ui32ClientCacheOpSeqNum		int32
+	ui32PrepareCount		len[pui32ClientUpdateCount, int32]
+	pui32ClientUpdateCount		ptr[in, array[int32]]
+	phUpdateUFOSyncPrimBlock	ptr[in, array[ptr[in, array[IMG_HANDLE]]]]
+	pui32UpdateSyncOffset		ptr[in, array[ptr[in, array[int32]]]]
+	pui32UpdateValue		ptr[in, array[ptr[in, array[int32]]]]
+	hCheckFenceFD			PVRSRV_FENCE
+	h2DUpdateTimeline		PVRSRV_TIMELINE
+	h3DUpdateTimeline		PVRSRV_TIMELINE
+	puiUpdateFenceName		ptr[in, array[int8]]
+	pui32CommandSize		ptr[in, array[int32]]
+	pui8FWCommand			ptr[in, array[ptr[in, array[int8]]]]
+	pui32TQPrepareFlags		ptr[in, array[int32]]
```

Comment:
The possible values in this array might be TQ_PREP_FLAGS_COMMAND_3D, TQ_PREP_FLAGS_PDUMPCONTINUOUS, TQ_PREP_FLAGS_COMMAND_2D etc.

Reply:
Fixed.